server/src/api/bill_exec.py

```diff
@@ -24,10 +24,41 @@ def __init__(self, database):
 
     @cherrypy.expose
     def create_bill(self):
-        """modifies transaction on database
-        Check database structure doc
+        """creates a bill and adds to 
+            the database accordingly
         
-        Arguments:
-            json -- check api doc for json format
+        JSON Arguments:
+            title -- user defined name of bill
+            payer -- name of person who fronted bill
+            total -- total bill amount
+            outstanding_payments -- list of outstanding payments
+                person -- name of payer
+                amount -- amount outstanding
+        JSON Returns:
+            success -- bool success of fail
+            message (optional) - error message if applicable
         """
-        return "3"
\ No newline at end of file
+        JSON_object = json.loads(cherrypy.request.body.read().decode('utf-8'))
+        title = JSON_object.get("title")
+        payer = JSON_object.get("payer")
+        total = JSON_object.get("total")
+        outstanding_payments = JSON_object.get("outstanding_payments")
+        
+        error_msg = ""
+        success = True
```

Comment:
default success should be false. right now I think a KeyError will return success : true